### PR TITLE
Added feature for demo account usage

### LINKED ISSUE #75: [FEATURE] Easy demo access
### What?

Currently, users have to copy and paste the credentials to enter the demo. It would be better to have a button on the landing page to access the demo account directly

### Why?

The current login system for the demo isn't user-friendly; it's too much work

### How?

1. **Delete the information on the sign-in page:**  
   <p align="center">
       image
   </p>

2. **Add a button on the landing page** that automatically redirects to the dashboard with the demo account login.

### PR DESCRIPTION
**Pull request by Villain45**
closes #75 
image

### CODE REVIEW

File: src/app/signin/auth-ui.tsx [hunk 1]
```diff
@@ -40,13 +50,10 @@ export default function AuthUI() {
...
         Would you like to give it a try? Feel free to use our demo account 😉
-        <p>
-          email: <code className="text-sm ">tiffs_purview0r@icloud.com</code>
-        </p>
-        <p>
-          password: <code className="text-sm ">qwerty</code>
-        </p>
       </div>
+      <Button variant={"default"} onClick={setDemoEmailandPassword}>
+        Use Demo Creds
```
Comment (manuelalferez) [inline src/app/signin/auth-ui.tsx:55]:
```suggestion
        Access Demo Account 
```

**Comment by manuelalferez** [comment]
@Villain45 is it possible that when the user click into the Demo button, the Sign in button is automatically clicked? 
<p align="center">
    image
</p>
In this way, the user just access to the account. More clean

**Comment by manuelalferez** [comment]
Nice approach, as usual  ;)

**Comment by Villain45** [comment]
> @Villain45 is it possible that when the user click into the Demo button, the Sign in button is automatically clicked?
> 
> image
> 
> In this way, the user just access to the account. More clean

Ok will add this as well then

**Comment by manuelalferez** [comment]
@Villain45 thanks!

**Comment by Villain45** [comment]
@manuelalferez Fine will create new branch

**Comment by manuelalferez** [comment]
Thanks for the feature! I will merge it in the other PR. It should count for your hacktoberfest as well, since there is commit that are yours. Let me know if it's like that, otherwise I will find a solution for that.

**Comment by Villain45** [comment]
@manuelalferez It's totally fine if it doesn't count

**Pull request closed without merging**